.github/workflows/publish.yml

```diff
@@ -234,6 +234,11 @@ jobs:
           git checkout -b publish/$GITHUB_REF_NAME
           npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
           pnpm nx-release --local=false $GITHUB_REF_NAME
+      - name: Trigger Docs Release
+        if: "!github.event.release.prerelease"
```

Comment:
I'm not actually familiar with this approach of wrapping the expression in a string, is this something you've seen documented somewhere?

Reply:
Updated.  It could be an old syntax.  I'm not sure if it still works, but even if it does, it is better to keep the syntax consistent.

Reply:
I got it from here: https://github.com/orgs/community/discussions/26281#discussioncomment-3251177
Original comment from 2019, but comments using it from march 2023